coral/src/app/features/approvals/components/RequestDeclineModal.test.tsx

```diff
@@ -124,8 +124,7 @@ describe("RequestDeclineModal.test", () => {
     });
 
     it("shows error when entering a too long message", async () => {
-      const tooLong =
-        "Quisque commodo aliquam tristique. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Sed ornare turpis ac cursus vulputate. Morbi auctor sodales porttitor. Mauris placerat ante id facilisis vehicula. Pellentesque ornare quis massa elementum auctor. Suspendisse potenti. Phasellus dignissim sit amet risus vitae aliquet. Vivamus at dolor vehicula, placerat odio sit amet, imperdiet enim. Donec scelerisque pretium metus ut dignissim. Morbi posuere tortor in cursus porttitor. Maecenas a diam ut urna mattis convallis a vel ligula.";
+      const tooLong = "x".repeat(301);
```

Comment:
React 19 uses automatic concurrency and startTransition more aggressively, so UI updates (including eg. state triggered by typing) can be batched or deferred. user.type has a slight delay by default to simulate typing - this is not noticeable locally but on CI tests can be slower. Added delay: 0 to hopefully solve this als well as updated the test string so it's more clear what "too long" means.